src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -253,6 +253,10 @@ To avoid these issues, set wait conditions that will pause the script until the
 
 You can also manually log monitor results to the [script log](/docs/synthetics/new-relic-synthetics/using-monitors/viewing-monitor-results#viewing-logs). Use logging to troubleshoot a script: to discover which step of your script is failing, include a log function along with each key step in your script.
 
+<Callout variant="important">
+  The Maximum Log length is 50000 Bytes. Script logs larger than 50000 Bytes are truncated.
```

Comment:
```suggestion
  The maximum log length is 50000 Bytes. Script logs larger than 50000 Bytes are truncated.
```

Reply:
This seems like a capitalization error, but can you confirm real quick before I merge? Thanks!

Reply:
Yup it was.